src/app/shared/components/bank-account-cards/bank-account-cards.component.ts

```
@@ -28,13 +28,17 @@
     },
   };
 
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)